tests/CrossOsAgnosticEqualsTrait.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Tests\Qossmic\Deptrac;
+
+use SebastianBergmann\Comparator\Factory;
+use staabm\PHPUnitCrossOs\Comparator\CrossOsAgnosticStringComparator;
+
+trait CrossOsAgnosticEqualsTrait {
```

Comment:
just added this small test-utility trait which is based on a [small phpunit cross-os helper package](https://github.com/staabm/phpunit-cross-os).

this trait helps us that we don't need to sprinkle `PHP_EOL`, `\n` or similar windows special cases over the test-classes.
the trait enables phpunits native `assertEqual*` assertion to be forgiving about `DIRECTORY_SEPARATOR` or `PHP_EOL` differences.